condor_watch_q.py

```diff
@@ -216,6 +223,24 @@ def parse_args():
     return args
 
 
+def parse_unknown_args(unknown):
+
+    for unknown_arg in unknown:
+        print("Unknown command.", end=" ")
```

Comment:
Instead of printing twice and using `end`, let's build up the full message incrementally and then print it out all at once. Also, we should print this message to stderr, not stdout.